scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -846,7 +851,20 @@ private Address convertAddressToRamSpace(Address address) throws Exception {
 			if (address == null) {
 				return null;
 			}
-			return ram_space.getAddress(address.toString(false));
+
+			try {
+				// If already in RAM space, just return it
+				if (address.getAddressSpace().getName().equals("ram")) {
```

Comment:
I like this, and I do still have a question (something I was going to work back around to answering myself, until I got a bit bogged down figuring out Ghidra test jar sourcing, but you probably know the answer already anyway given this change) - is there any way we can primarily work in the default address space for a given program? Why do we want RAM space at all given not everything (Thumb/EABI, so far) seems to not be compatible?

Reply:
The function gets called only for const space. We have this because many instances data references are recovered as ref to `const` varnode and we need corresponding ram space to identify if it actually refers address like `DATA_0x0`. 

I updated a check where if its not const space return the address. We don't want someone to call this function for `unique` or `register` space as well.

Reply:
Got it thank you - yeah that was not clear to me at all, I appreciate the update